build.xml

```diff
@@ -74,7 +74,7 @@
     <javac compiler="org.eclipse.jdt.core.JDTCompilerAdapter" source="1.8"
          target="1.8" srcdir="${src.dir}:${test.dir}" destdir="${build.classes.dir}" debug="true" debuglevel="lines,vars,source" includeantruntime="false">
       <classpath refid="classpath.base"/>
-      <compilerarg value="-warn:+allJavadoc" />
+      <compilerarg value="-warn:+allJavadoc,+invalidJavadocVisibility(public),+missingJavadocTagsVisibility(public)" />
```

Comment:
maybe it should be -invalidJavadocVisibility(public) 

..."specifying the given option with "+" will enable more warnings than the default, and specifying "-" disables some that are enabled by default"